src/_assets/css/_subscribe.scss

```diff
@@ -0,0 +1,160 @@
+.subscribe {
+  $required-input-color: #C53929;
+  $form-background-color: #E1EAF2;
+  $fields-border-color: #D8D8D8;
+  color: $site-color-paragraph;
+
+  &-container {
+    margin: 0 auto;
+    padding: 60px 0 0 0;
+
+    .heading {
+      color: $site-color-title;
+      font-size: 32px;
+      letter-spacing: -0.4px;
+      margin-bottom: 16px;
+      text-align: center;
+    }
+
+    .subheading {
+      margin-bottom: 40px;
+      text-align: center;
+    }
+
+    @include media-breakpoint-up(md) {
+      padding-top: 80px;
+
+      .heading {
+        font-size: 48px;
+        letter-spacing: -0.6px;
+      }
+
+      .subheading {
+        margin-bottom: 60px;
+      }
+    }
+
+    .subscribe-container__form {
+      background-color: $form-background-color;
+      border-radius: 4px;
+      margin: 0 -25px;
+      padding: 40px 25px 120px;
+
+      .subscribe-input {
+        height: 48px;
+      }
+
+      .subscribe-select {
+        display: block;
+        height: 36px;
+        font-size: 14px;
+        margin-bottom: 20px;
+        max-width: 242px;
+        padding: 8px;
+        text-transform: uppercase;
+      }
+
+      .subscribe-checkbox {
+        margin-right: 5px;
+      }
+
+      .subscribe-input, .subscribe-select {
+        border-color: $fields-border-color;
+        color: $site-color-paragraph;
+      }
+
+      .field-required {
+        color: $required-input-color;
+      }
+
+      .subscribe-form-group {
+        margin-bottom: 32px;
+      }
+
+      .form-label {
+        font-family: "Google Sans", "Roboto", sans-serif;
```

Comment:
Shouldn't this font be used for the entire page?

Reply:
No, there are some other texts where the text should be `Roboto`, if we set `Google Sans` for the entire page, we will have to go for each text where the font-family should be `Roboto` and set it in that way, I think this way is easier.